packages/contact-center/station-login/src/station-login/station-login.types.ts

```diff
@@ -8,22 +10,66 @@ export interface IStationLoginProps {
   name: string;
 
   /**
-   * The current login state of the station.
+   * Webex instance.
    */
-  loginState: any;
+  webex: Webex;
 
   /**
-   * Function to set the login state of the station.
+   * Array of the team IDs that agent belongs to
    */
-  setLoginState: any;
+  teams: Team[];
+
+   /**
+   * Station login options available for the agent
+   */
+  loginOptions: string[];
+
+  /**
+   *  Request Param object to be sent to the SDK station login method
+   */
+  loginReqParam: AgentLogin
```

Comment:
; seems to be missing? same for the functions below too

Reply:
added